vms/volunteer/templates/volunteer/edit.html

```diff
@@ -229,10 +231,10 @@
                     </div>
                 </div>
                 {% if form.unlisted_organization.errors %}
-                    <div class="form-group has-error">
+                    <div id="div_id_unlisted_organization" class="form-group has-error">
                         <label class="col-md-2 control-label">{% trans "If your organization is not listed, please provide it here:" %}</label>
-                        <div class="col-md-8">
-                            <input class="form-control" type="text" placeholder="{% blocktrans %}Organization{% endblocktrans %}" name="unlisted_organization" value="{% if form.unlisted_organization.value %}{{ form.unlisted_organization.value }}{% endif %}">
+                        <div id="div_id_unlisted_organization" class="col-md-8">
+                            <input id="id_unlisted_organization" class="form-control" type="text" placeholder="{% blocktrans %}Organization{% endblocktrans %}" name="unlisted_organization" value="{% if form.unlisted_organization.value %}{{ form.unlisted_organization.value }}{% endif %}">

```

Comment:
ideally, id should be unique - and if there is a need for you to do this, there might be something wrong with the original design - and we might want to change it ? 

https://www.w3.org/TR/html4/struct/global.html 7.5.2

`div_id_unlisted_organization`

Reply:
@tonythomas01 input id is not needed. Thus removed them. Please review.